docs/cluster-configuration.md

```diff
@@ -130,6 +130,12 @@ or new features that are not enabled by default.
 To enable a feature, add its name to the `featureGates` list and set it to `true`. Missing or `false` feature gates
 disables the feature.
 
+### downwardMetrics Feature Gate
+Set the `downwardMetrics` feature gate in order to allow exposing a limited set of VM and host metrics to the guest.
+The format is compatible with [vhostmd](https://github.com/vhostmd/vhostmd).
+These metrics allow third-parties diagnosing issues.
+DownwardMetrics may be exposed to the guest through a `volume` or a `virtio-serial port`.
```

Comment:
Please document the upgrade strategy (in the past it was always on)

Reply:
Done.